package.json

```diff
@@ -52,14 +52,24 @@
 	"license": "Apache-2.0",
 	"dependencies": {
 		"@babel/parser": "^7.2.2",
+		"@types/d3-array": "^2.0.0",
+		"@types/d3-axis": "^1.0.12",
+		"@types/d3-drag": "^1.2.3",
+		"@types/d3-selection": "^1.4.2",
+		"@types/d3-transition": "^1.1.6",
 		"@types/hoist-non-react-statics": "^3.3.1",
 		"classnames": "^2.2.3",
 		"create-react-class": "^15.6.2",
+		"d3-array": "^2.4.0",
```

Comment:
Is there any reason we have all of these d3 packages. How about just importing d3 (npm install d3). I believe you can still just import in in the file things like d3-axis. https://www.npmjs.com/package/d3

Reply:
That makes sense, I think the initial idea was that is how it was being done here (`package.json` was already including some d3 packages but not all) probably to prevent including things we don't need. But yeah with this component specifically we are using more d3 packages. Changing this however would require changes to anything in lucid which imports a d3 package (change being `import ... from 'd3-specificPackage'` -> `import ... from 'd3'`)

Reply:
It seems okay with me to follow the pattern that's there of importing individual packages. We could do a separate PR for d3.

Reply:
> Scope-wise this feels like a separate PR if we would like to only import d3 as a whole.
agreed

also, importing separate d3 packages makes sense to me since we're clear about the dependency and not importing an umbrella package, though it makes it easier, it isn't clear about what part we're using and has more than we need. we don't use the dom part of d3, I don't think we should pull in d3.